zkevm-circuits/src/evm_circuit.rs

```diff
@@ -665,6 +659,7 @@ impl<F: FieldExt> EvmCircuit<F> {
                     .map(|(expr, column)| {
                         (
                             expr.clone(),
+                            // I think we have some problem here
                             meta.query_advice(*column, Rotation::cur()),
```

Comment:
In this line, we are doing a lookup to an `advice` column. The new API seems not to allow us to do that anymore. Would like to hear feedback from @han0110 and @therealyingtong.